web/package.json

```diff
@@ -35,10 +35,13 @@
   },
   "homepage": "https://github.com/18F/cms-hitech-apd#readme",
   "dependencies": {
+    "history": "^4.7.2",
     "prop-types": "^15.6.0",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
     "react-redux": "^5.0.6",
+    "react-router-dom": "^4.2.2",
+    "react-router-redux": "^5.0.0-alpha.9",
```

Comment:
Should we explicitly declare a dependency on `react-router`?  I know we'll get it implicitly and maybe that's better - fewer direct dependency versions to manage, and let the downstream ones shake themselves out.  🤔   Now I've typed these words and I think the implicit dependency is better. 😛

Reply:
hehe, cool, i like the implicity (that really should be a word!) in this case, and it's also how they do in their docs / guides